ThunderCloud/NavigationController.swift

```diff
@@ -190,6 +190,9 @@ public extension UINavigationController {
 			UIApplication.shared.open(url, options: [:], completionHandler: nil)
 			
 		} else {
+            guard let rightMostNavigationController = UIApplication.shared.keyWindow?.rightMostNavigationController else {
```

Comment:
Should we be falling back to `self` here?

Reply:
@simonmitchell resolved in 0677f8a8cab6c9fe6b2e2d17943394fb19e23f39 :D